wremnants/CardTool.py

```diff
@@ -854,13 +870,24 @@ def fillCardWithSyst(self, syst):
         systnamesPruned = [s for s in systNames if not self.isExcludedNuisance(s)]
         systNames = systnamesPruned[:]
         for chan in self.channels:
-            for systname in systNames:
-                shape = "shape" if not systInfo["noConstraint"] else "shapeNoConstraint"
-                # do not write systs which should only apply to other charge, to simplify card
-                self.cardContent[chan] += f"{systname.ljust(self.spacing)} {shape.ljust(self.systTypeSpacing)} {''.join(include_chan[chan])}\n"
+            systNamesChan = [x.replace("CHANNEL",chan) for x in systNames]
+            for systname in systNamesChan:
+                systShape = shape
+                include_line = include_chan[chan]
+                # some customization based on nuisance
+                if systInfo["customizeNuisance"]:
+                    for regexpCustom in systInfo["customizeNuisance"]:
+                        if re.match(regexpCustom, systname):
+                            keys = list(systInfo["customizeNuisance"][regexpCustom].keys())
+                            if "scale" in keys:
+                                include_line = [(str(systInfo["customizeNuisance"][regexpCustom]["scale"]) if x in procs else "-").ljust(self.procColumnsSpacing) for x in nondata_chan[chan]]
+                            if "shape" in keys:
+                                systShape = systInfo["customizeNuisance"][regexpCustom]["shape"]
+                ## end of customization
```

Comment:
I'm not a fan of comments like this because they can end up in the middle of nowhere if code gets moved around

Reply:
I have to remove them, it is a leftover from when I was messing up a bit in the code